### DISCUSSION TITLE
Does the Pocketbook build work on the new Pocketbook Era?

### DISCUSSION THREAD

**Discussion by MultiKoopa** [koreader/koreader#9347] [Q&A] [answered]
Or is the Era too different, and the program needs to be modified to be compatible with it?

**Comment by Frenzie** [comment]
It would require adding a new device definition (if you run it you'll find out at least part of it), besides that there's no way to tell in advance.

**Comment by ElimGarak1** [comment, marked as answer]
From my new Pocketbook Era (PB700) 

![Era](https://user-images.githubusercontent.com/84080014/180596871-ea3fd619-f8c8-4806-90d9-1c8da911b57b.jpg)

![PB 700_1](https://user-images.githubusercontent.com/84080014/180597172-35404568-dab6-488a-9a3b-4075b882e2cb.jpg)

![PB 700_2](https://user-images.githubusercontent.com/84080014/180597180-5fd20112-0dc0-4de7-a8b6-23eece514cbb.jpg)

\applications\koreader\frontend\device\pocketbook [PB 700 new device.lua.zip](https://github.com/koreader/koreader/files/9173393/PB.700.new.device.lua.zip)

KOReader [crash.log](https://github.com/koreader/koreader/files/9173410/crash.log)

**Reply by Frenzie** [reply]
Would you like to submit a PR? :-)

**Reply by ElimGarak1** [reply]
I just wanted to be able to run KOReader on the PB 700 as well.

**Reply by MultiKoopa** [reply]
Will KOReader be updated to support the Era?

**Reply by Frenzie** [reply]
Yes, @ElimGarak1 you're sure you don't want to put it in a PR?

**Reply by ElimGarak1** [reply]
https://github.com/koreader/koreader/pull/9365 ;-)

**Reply by zmixnyc** [reply]
2022.07 seems to work pretty well on my PB700.  The second day I had a strange bug where the orientation would invert after a page turn. Re-booting cleared it up, though.
